pkg/image/api/validation/validation.go

```diff
@@ -10,13 +11,21 @@ import (
 	"github.com/openshift/origin/pkg/image/api"
 )
 
+func MinimalNameValidation(name string, prefix bool) (bool, string) {
```

Comment:
I see a lot are using ValidatePodName and others are using custom funcs that handle % and / and other characters. It'd be nice to have a single common func that all validations could use to check the bare minimum, and then custom logic could be added after that. But I don't think it's required for this PR.